ADO.NET/OAuthSampleApp/README.md

```diff
@@ -0,0 +1,29 @@
+## Overview
+
+The Vertica ADO.NET driver now allows connecting through OAuth access tokens. 
+
+This sample application was built in .NET 6.0 and can be run in .NET Framework 4.6+ or .NET Core 3.1+.
+
+# Prerequisites
+
+For information on installing .NET, starting a Vertica docker container, and how to reference a NuGet package, see the README.md in the ADO.NET base directory.
+
+# Using OAuth to login
+
+You must first configure the Vertica server for OAuth as described here: https://docs.vertica.com/24.1.x/en/security-and-authentication/client-authentication/oauth-2-0-authentication/configuring-oauth-authentication/.
```

Comment:
Some preference for "latest" in URL instead of "24.1.x" so that we don't have to update the URL or have a link to a previous release's docs after each release.

Reply:
Good idea.